Dockerfile

```diff
@@ -1,13 +1,12 @@
-FROM openjdk:8-jre
+FROM hmcts/cnp-java-base:openjdk-jre-8-slim-stretch-1.0
 
-COPY build/bootScripts/spring-boot-template /opt/app/bin/
+ENV APP spring-boot-template.jar
+ENV APPLICATION_TOTAL_MEMORY 1024M
+ENV APPLICATION_SIZE_ON_DISK_IN_MB 41
+ENV JAVA_OPTS ""
 
-COPY build/libs/spring-boot-template.jar /opt/app/lib/
+COPY build/libs/$APP /opt/app/
```

Comment:
$APP is jar name ? may be at least put a comment to replace it so that is not missed.

Reply:
the init script should automatically do that. after #63 should fix issues some of us were facing. want to try out to verify too? would be great to know